test/extended/router/certs.go

```diff
@@ -101,7 +101,7 @@ u3YLAbyW/lHhOCiZu2iAI8AbmXem9lW6Tr7p/97s0w==
 
 	g.When("FIPS is enabled", func() {
 		g.Describe("the HAProxy router", func() {
-			g.It("should not work when configured with a 1024-bit RSA key", func() {
+			g.It("should not work when configured with a 1024-bit RSA key [apigroup:template.openshift.io]", func() {
```

Comment:
This is not good.  Please mark for followup, and probably even a bug.  The router needs to work on microshift.